src/emulator/dataconnectEmulator.ts

```diff
@@ -35,24 +35,27 @@ export class DataConnectEmulator implements EmulatorInstance {
 
   async start(): Promise<void> {
     this.logger.log("DEBUG", `Using Postgres connection string: ${this.getLocalConectionString()}`);
-    const info = await DataConnectEmulator.build({ configDir: this.args.configDir });
-    if (requiresVector(info.metadata)) {
-      if (Constants.isDemoProject(this.args.projectId)) {
-        this.logger.logLabeled(
-          "WARN",
-          "Data Connect",
-          "Detected a 'demo-' project, but vector embeddings require a real project. Operations that use vector_embed will fail.",
-        );
-      } else {
-        this.logger.logLabeled(
-          "WARN",
-          "Data Connect",
-          "Operations that use vector_embed will make calls to production Vertex AI",
-        );
+    try {
+      const info = await DataConnectEmulator.build({ configDir: this.args.configDir });
+      if (requiresVector(info.metadata)) {
+        if (Constants.isDemoProject(this.args.projectId)) {
+          this.logger.logLabeled(
+            "WARN",
+            "Data Connect",
+            "Detected a 'demo-' project, but vector embeddings require a real project. Operations that use vector_embed will fail.",
+          );
+        } else {
+          this.logger.logLabeled(
+            "WARN",
+            "Data Connect",
+            "Operations that use vector_embed will make calls to production Vertex AI",
+          );
+        }
       }
+    } catch (err: any) {
+      this.logger.log("DEBUG", `'fdc build' failed with error: ${err.message}`);
     }
     return start(Emulators.DATACONNECT, {
-      ...this.args,
```

Comment:
While testing, I noticed this was leading to a bunch of 'Ignored unrecognized arg' debug logs, so I cleaned it up.

Reply:
I think we should keep `auto_download`. It's probably not needed in this context because `.build` happens to download the binary, but this probably won't survive a refactor or two.